src/shared/interfaces/Event.ts

```diff
@@ -0,0 +1,25 @@
+import { User } from './User';
+
+export interface Event {
```

Comment:
Double check the interfaces with the app and the back end. There are mistakes, like "likes" is optional and etc. Check them all